GameServerLib/Content/NavGrid.cs

```diff
@@ -268,6 +341,23 @@ public int GetCellIndex(float x, float z)
 
             return -1;
         }
+        private List<NavGridCell> GetCellNeighbors(NavGridCell cell)
+        {
+            var x = cell.X;
+            var y = cell.Y;
+            List<NavGridCell> neighbors = new List<NavGridCell>();
+            for (int dirY = -1; dirY <= 1; dirY++)
+            {
+                for (int dirX = -1; dirX <= 1; dirX++)
```

Comment:
Damn this is way better than `List<NavGridCell> neighbors = { map[x-1,y], map[x, y-1], ... }` how did I never thought about this lmao